roundtables/week12/day-5-inspiration-board-luhn.md

```diff
@@ -0,0 +1,33 @@
+# Inspiration Board and Exercism: Luhn
+
+Today we are going to be discussing the Inspiration Board project and the Luhn Exercism in a round table fashion. For this discussion, think of your instructor as a mediator and guide, rather than as a lecturer.
+
+For this activity, you are encouraged to do the following:
+
+* Whenever possible, you should use voice.
+* You should pose questions to the group, not the instructor.
+* Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+* Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try)
+* When discussing code, you are encouraged to use VS Code Liveshare, and or screen sharing through zoom.
+* Take notes!
+
+## Materials
+[Inspiration Board](https://github.com/Ada-C13/inspiration-board)
+
+[Exercism (JavaScript Track)](https://exercism.io/my/tracks/javascript)
+
+## Prompts
+You are encouraged to copy these questions into a document and take notes during the discussion.
+
+### Inspiration Board
+1. What are all the components in this project? How do the components relate to each other? 
```

Comment:
Maybe ask where we need to make the API request?

Reply:
Does this feel like a helpful question even though its explicitly stated in the wave 2 instructions?

Reply:
I could see that being useful to get people comfortable answering questions with an easy one but I'm inclined to leave it off because we already have more questions here than most roundtable discussion guides have had.